### PR TITLE
trigger logger event and and retry for git ops added

### PR DESCRIPTION
**Pull request by lfpratik**

### CODE REVIEW

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm please ping me when merged, because I will have to rebuild the base image.

File: docker-images/base/patch/perceval-trigger-error-event.py.diff [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
++
++
++def get_smtp_handler():
++    return SDSSMTPHandler(
++        mailhost=('smtp.gmail.com', 465),
++        fromaddr='dev.analytics.error.bot@gmail.com',
++        toaddrs=['pratikk@proximabiz.com', 'sgupta@linuxfoundation.org',
++                 'fghiasy@linuxfoundation.org'],
++        subject='SDS Error - log',
++        credentials=('dev.analytics.error.bot@gmail.com', 'Mobile@311!'),
```
Comment (lukaszgryglicki) [inline docker-images/base/patch/perceval-trigger-error-event.py.diff:273]: Remove the sensitive data from a public PR.

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
Remove the sensitive data from a public repo.

File: docker-images/base/patch/perceval-trigger-error-event.py.diff (hunk 1 above)
Comment (lfpratik) [reply docker-images/base/patch/perceval-trigger-error-event.py.diff:273]: @lukaszgryglicki, I will do it. Let me know the environment variable I will pull those value from their.

Comment (lfpratik) [reply docker-images/base/patch/perceval-trigger-error-event.py.diff:273]: I will update the same for ALL PR

**Comment by lfpratik** [comment]
@lukaszgryglicki, can you review this PR

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**